go.mod

```diff
@@ -102,9 +105,10 @@ require (
 	github.com/rabbitmq/amqp091-go v1.4.0
 	github.com/rcrowley/go-metrics v0.0.0-20201227073835-cf1acfcdf475
 	github.com/rickb777/date v1.17.0
+	github.com/rivo/uniseg v0.3.4 // indirect
```

Comment:
I'm not really sure why this got updated when I ran `go get -u github.com/segmentio/parquet-go`, but if #1307 is good to go, then we can drop Go 1.17 support and the CI build will succeed...

```shell
> go mod why github.com/rivo/uniseg
# github.com/rivo/uniseg
github.com/benthosdev/benthos/v4/internal/impl/parquet
github.com/segmentio/parquet-go
github.com/olekukonko/tablewriter
github.com/mattn/go-runewidth
github.com/rivo/uniseg
```
If I look in https://github.com/olekukonko/tablewriter/blob/master/go.mod, it hasn't been updated in over a year...